### DISCUSSION TITLE
MIES Supported Amplifiers

### DISCUSSION THREAD

**Discussion by mvg260** [AllenInstitute/MIES#1435] [Q&A] [answered]
Hello, is it possible to use MIES software with Axopatch 200B amplifiers? Obviously software control of amplifier settings & telegraphing would not be possible, but are these required for MIES to work?

**Comment by t-b** [comment, marked as answer]
Right now MIES only supports the Molecular Devices 700B as outlined in https://github.com/AllenInstitute/MIES#supported-hardware.

Is that hardware still for sale? The manual at https://mdc.custhelp.com/euf/assets/content/Axopatch_200B_Manual_RevE2.pdf is from 1999. From glancing over the manual I also don't see a way of programmatically setting parameters on the amplifier. Do you know how that would work?

You can in general use MIES without amplifier. Turn off in the DAEphys panel `Settings-Amplifier->Require Amplifier` and you are good to go.

**Reply by mvg260** [reply]
Thanks for your reply Thomas. 
I’m not sure if the 200Bs are still made but would guess they’re still among the most widely used patch-clamp amplifiers worldwide. They indeed have no software interfacing capabilities, so all functions like switching modes, adjusting gains, capacitance settings and bridge balancing would have to be done manually by the experimenter turning knobs on the amplifier. 

I understand that MIES is developed with the 700B amp in mind, I’m just really keen to use MIES and so was hoping there was a way to use it simply as acquisition software (like pClamp or WinWCP), and not use the features that require amplifier control/communication. 
If I turn off the ‘require amplifier’ setting, can I provide MIES with the information about gains and recording mode to acquire a correctly scaled signal? 

Thanks

**Reply by t-b** [reply]
> If I turn off the ‘require amplifier’ setting, can I provide MIES with the information about gains and recording mode to acquire a correctly scaled signal?

Yes you can! When you don't use an amplifier you have to set the gain and scaling factors directly in the DA and AD tab.

**Reply by mvg260** [reply]
Oh this is great news, thanks so much!
